cps/web.py

```diff
@@ -1568,7 +1569,25 @@ def read_book(book_id, book_format):
             if book_format.lower() == fileExt:
                 entries = calibre_db.get_filtered_book(book_id)
                 log.debug("Start video watching for %d", book_id)
-                return serve_book.__closure__[0].cell_contents(book_id, book_format.lower(), anyname="")
+                video_path = os.path.join(config.config_calibre_dir, book.path)
+                # look into video_path for any file with webm or mp4 extension
+                video_file = ""
+                for file in os.listdir(video_path):
+                    if file.endswith(".webm") or file.endswith(".mp4"):
+                        video_file = os.path.join(video_path, file)
+                        # align with nginx path
```

Comment:
```suggestion
                        # align with nginx path (e.g. "books-direct") specified on Lines ~21 and ~25 of:
                        # https://github.com/iiab/calibre-web/blob/master/scripts/calibre-web-nginx.conf
```

Reply:
@deldesir can you take the above suggestion seriously — or if not explain why?

In lieu of soft-coding (or posting constants on top of files, as is traditional to make software more maintainable...) please at least consider clear documentation explaining how to maintain de facto global constants.